README.md

```diff
@@ -8,3 +8,15 @@
 A simple calculator application built using the Fyne toolkit.
 
 ![](img/calc-dark.png) &nbsp; ![](img/calc-light.png)
+
+# Installation instructions
+Install Fyne:
+```go get fyne.io/fyne/v2/cmd/fyne```
+
+Install resources for this application:
+go to folder /calculator
```

Comment:
I don't think the folder should be /calculator as it will not have been downloaded to the root.

Reply:
I do not understand. My understanding is that i have to generate a file within the project. 

I cloned the repo, hence the /calculator.

Reply:
The folder should be `$GOPATH/src/github.com/fyne-io/calculator` most likely. The folder `/calculator` would be a folder at the root of your file system and would not be the place where go installs it to.